torchao/quantization/subclass.py

```diff
@@ -36,6 +38,30 @@
 
 aten = torch.ops.aten
 
+def _aqt_is_int8(aqt):
+    """Check if an AffineQuantizedTensor is int8 quantized Tensor"""
+    return (
+        aqt.int_data.dtype == torch.int8 and
+        aqt.quant_min is None or aqt.quant_min == -128 and
+        aqt.quant_max is None or aqt.quant_max == 127
+    )
+
+def _aqt_is_int8_reduce_range(aqt):
```

Comment:
So an aqt can be both int8 and int8 reduced_range if quant_min is None.

Should we require quant_min to always be defined? Is there an implied value when it's None?

Reply:
Also nit I'd use `reduced` instead of `reduce`, just because `reduce` is a verb. This doesn't matter quite that much since it's a private API for now.

Reply:
oh sorry, I think we want to remove `None` here